bodhi/client/__init__.py

```diff
@@ -337,6 +332,29 @@ def download(url, **kwargs):
                         click.echo("WARNING: download of {0} failed!".format(build['nvr']))
 
 
+def _get_notes(**kwargs):
+    """
+    If the user provides a --notes-file, _get_notes processes the contents of the notes-file. 
+    If the user does not provide a --notes-file, _get_notes() returns the notes from the kwargs. 
```

Comment:
There is a style test that is failing on these two lines due to trailing whitespace:

```
/home/vagrant/bodhi/bodhi/client/__init__.py:337:94: W291 trailing whitespace
/home/vagrant/bodhi/bodhi/client/__init__.py:338:97: W291 trailing whitespace
F.
======================================================================
FAIL: Enforce PEP-8 compliance on the codebase.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vagrant/bodhi/bodhi/tests/test_style.py", line 37, in test_code_with_flake8
    self.assertEqual(subprocess.call(flake8_command), 0)
AssertionError: 1 != 0
```

I think if you remove that whitespace we'll be good to merge this!